ImagineEngine.xcodeproj/project.pbxproj

```diff
@@ -925,10 +931,10 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++14";
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
-				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
@JohnSundell, please note that I've also removed **Code Signing Identity** and **Development Team** for macOS tests target to make it possible to run them. Let me know if you're OK with these changes.

Reply:
Totally, it shouldn't have been there to begin with - thanks 👍